datasets/requirements.txt

```diff
@@ -1,4 +1,12 @@
+anthropic==0.31.1
```

Comment:
Do we really need all of these packages? These reqs should ideally be very minimal since they need to be installed on the cluster and we try to avoid that as much as possible